$OpenBSD: patch-test_openssl_test_ssl_rb,v 1.2 2021/09/09 14:57:22 tb Exp $

Remove a pend that hides missing support for SSL_CTX_set_client_cert_cb()
in the TLSv1.3 stack.

Index: test/openssl/test_ssl.rb
--- test/openssl/test_ssl.rb.orig
+++ test/openssl/test_ssl.rb
@@ -344,8 +344,9 @@ class OpenSSL::TestSSL < OpenSSL::SSLTestCase
   end
 
   def test_client_ca
-    pend "LibreSSL 3.2 has broken client CA support" if libressl?(3, 2, 0)
-
+    if ENV['SKIP_CLIENT_CA_TEST']
+      pend "skip failing client CA test"
+    end
     ctx_proc = Proc.new do |ctx|
       ctx.client_ca = [@ca_cert]
     end
